cves/kernel/CVE-2021-29266.yml

```diff
@@ -235,10 +233,8 @@ interesting_commits:
       * Other commits that fixed a similar issue as this vulnerability
       * Anything else you find interesting.
   commits:
-  - commit:
-    note:
-  - commit:
-    note:
+  - commit: f6bbf0010ba004f5e90c7aefdebc0ee4bd3283b9
+    note: "This commit fixed a mistake made in the original VCC: 776f395004d8"
```

Comment:
Maybe add what the fix was? Was it also a pointer issue or something completely unrelated